configs/examples/config_alignn.yml

```diff
@@ -0,0 +1,95 @@
+
+trainer: property
+
```

Comment:
Can you please clean up these two config files and remove all of the unnecessary spacing/commented out items?